docs/tutorials/remote_client.md

```diff
@@ -62,7 +62,10 @@ Remote Podman uses SSH to communicate between the client and server. The remote
 ```
 ssh-keygen -t ed25519
 ```
-Your public key by default should be in your home directory under ~/.ssh/id_ed25519.pub. You then need to copy the contents of id_ed25519.pub and append it into  ~/.ssh/authorized_keys on the Linux  server. You can automate this using ssh-copy-id.
+Your public key by default should be in your home directory under ~/.ssh/id_ed25519.pub. You then need to copy the contents of id_ed25519.pub and append it into  ~/.ssh/authorized_keys on the Linux  server. You can automate this using ssh-copy-id:
```

Comment:
It's not always going to be `id_ed25519.pub` right?  Perhaps we can make it a generic `{MyPublicKey}` and then list an example for the first one ala:

```suggestion
Your public key by default should be in your home directory under ~/.ssh/{MyPublicKey}, where {MyPublicKey} would be named something like `id_ed25519.pub`.  You then need to copy the contents of id_ed25519.pub and append it into  ~/.ssh/authorized_keys on the Linux  server. You can automate this using ssh-copy-id:
```

Reply:
and then change id_ed25519.pub to {MyPublicKey} in the other places it's currently occuring.